CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the INCORE documents generated by Sphinx package will be
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Fixed
+- Getting start screenshot with wrong output key [#411](https://github.com/IN-CORE/pyincore/issues/411)
```

Comment:
Issue number doesn't look right - this links to pyincore.

Reply:
Ahh, there is a 411, but this needs to link to the one in docs.

Reply:
Ah i kept copying from the wrong entry. Fixed it.